shared/src/packages/users/libs/validation-schemas/common-auth-fields.ts

```diff
@@ -0,0 +1,23 @@
+import joi from 'joi';
+
+import { UserValidationMessage } from '../enums/user-validation-message.enum.js';
+
+const commonUserFields = {
```

Comment:
Let's remove this and use separate schemas

Reply:
But I was told to reuse validation schema? Sign in/ sign up schemas are different, they just use this one cause of the common inputs. Or did I understood wrongly what changes I was supposed to do?

Reply:
There are different approaches, but I would prefer to split them, because we will end up creating a lot of shared validation schemas, but when will need to have different validation, for the same field we will need either change a lot or override this field which will add complexity to the code. It's ok in some case, but for sign-up sign-in it's usual thing to have different validation, that's why it's better to split them

Reply:
if you are referring to this comment https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/48#discussion_r1301949396, I mean that it should be in shared folder so we can reuse them on backend/frontend, but they should be splitted